containers/allan_dip_bot/run_bot.py

```diff
@@ -283,6 +298,12 @@ def main() -> None:
     sleep_delay: bool = args.no_sleep_delay
     discount_factor: float = args.discount_factor
     outdir: Optional[Path] = args.outdir
+    if args.aggressiveness == "M":
+        aggressiveness = Aggressiveness.moderate
+    elif args.aggressiveness == "A":
+        aggressiveness = Aggressiveness.aggressive
+    elif args.aggressiveness == "F":
+        aggressiveness = Aggressiveness.friendly
```

Comment:
This code needs to be changed to use the new names, preferably with `.value` instead of string literals.

In addition, please add the proper type for the first declaration of the `aggressiveness` variable.

Reply:
done